### ISSUE TITLE
Linode Cli is it possible to get linode created date ?

### ISSUE THREAD

**Issue by yspanchal** [linode/cli#50]
Linode Cli is it possible to get linode created date ?

linode -o linode -a show -l label-1

**Comment by felicianotech** [comment]
Doesn't look like it to me, but that information is available via the API: https://www.linode.com/api/linode/linode.list

**Comment by yspanchal** [comment]
@felicianotech created pull request to add linode create timestamp https://github.com/linode/cli/pull/51

**Comment by felicianotech** [comment]
@yspanchal And I saw it was merged. Awesome stuff. Don't you love FOSS? =)

**Comment by displague** [comment]
Thanks for the feedback, @yspanchal and @felicianotech.

This cli is deprecated in favor of https://github.com/linode/linode-cli (`pip install linode-cli`) which does offer the ability to list Linode instances with their creation date:

```
$ linode-cli --format label,created linodes list
┌──────────────────────────────┬─────────────────────┐
│ label                        │ created             │
├──────────────────────────────┼─────────────────────┤
│ lspecial                     │ 2018-08-11T14:45:15 │
│ linode10192810               │ 2018-09-10T23:56:11 │
│ tf_test_amused_skink-simple  │ 2018-09-20T19:35:55 │
│ tf_test_amused_skink-nginx-2 │ 2018-09-20T19:36:02 │
│ tf_test_amused_skink-nginx-1 │ 2018-09-20T19:36:03 │
│ tf_test_amused_skink-nginx-3 │ 2018-09-20T19:36:04 │
│ linode10448389               │ 2018-09-24T16:52:07 │
└──────────────────────────────┴─────────────────────┘
```

You may also be interested in `linode-cli linodes list --all` to see all of the data available for your Linode instances.

---

*Additionally closing this issue because @yspanchal and @fafadiatech contributed the fix to this in #51*
